roles/etcd/tasks/main.yml

```diff
@@ -37,6 +37,12 @@
     mode: 0700
   when: etcd_is_thirdparty | bool
 
+- name: Create configuration file in sysconfig
+  file:
+    path: /etc/sysconfig/etcd
+    state: touch
+  when: etcd_is_thirdparty | bool
```

Comment:
Actually this does not work for containerized environments, I had to remove the conditional respectively to change it to ```etcd_is_thirdparty | bool or etcd_is_containerized | bool```, otherwise the systemd service ```etcd_container``` did not start.